src/utils/plots/plotNames.js

```diff
@@ -0,0 +1,26 @@
+const plotType = {
```

Comment:
this contains more than plot names, I would name the file `contants.js`. It can stay in the plots folder but I believe it's best to place it directly inside the utils folder because otherwise it's easier to miss it, and the next developer might add his contacts somewhere else